### PR TITLE
Add instruction to do a recursive clone before attempting a build

### LINKED ISSUE #65: Mention recursive clone / submodule init in build instructions
While trying to build OTIO/Raven for the first time, me (and several other people I know) ran into the same issue: they did not do a recursive clone / initialize the submodules before running the build steps.

To do:
- Update the README to reflect the aforementioned instruction in the build steps

### PR DESCRIPTION
**Pull request by yaash45**
I made a quick PR to address this issue I created here #65. 

Fixes #65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ Linux (Ubuntu, or similar):
 - A recent version of CMake
   - You can get this via `sudo snap install cmake` or by downloading from https://cmake.org/download/
 
+__Note__: Before building, please ensure that you clone this project with the `-recursive` flag.
```
Comment (timlehr) [inline README.md:29]: Thank you! A little typo: `--recursive`

Comment (yaash45) [reply README.md]: Thanks for catching this! I've updated the line of the readme :)

**Review by jminor — APPROVED** [review]

**Comment by jminor** [comment]
I see there is also https://github.com/OpenTimelineIO/raven/pull/55 which takes a slightly different approach to fix the same issue. @ddouglas and @yaash45 would you two like to discuss pros/cons of either or both of these?

**Comment by yaash45** [comment]
> I see there is also #55 which takes a slightly different approach to fix the same issue. @ddouglas and @yaash45 would you two like to discuss pros/cons of either or both of these?

Sure thing! Thanks for linking the other PR. I like that #55 adds the submodule command into the build instructions. It would help to keep the submodules up-to-date. I think it needs some fixing though, the formatting is off.  I can leave a comment on it.

**Comment by jminor** [comment]
If @ddouglas isn't available right now (that PR was from a month ago) then you can fold that change into yours @yaash45 and then we can add a "Co-Authored-By" to the squashed commit when we merge the PR, so you both get credit for the contribution.

https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Comment by yaash45** [comment]
> If @ddouglas isn't available right now (that PR was from a month ago) then you can fold that change into yours @yaash45 and then we can add a "Co-authored-by:" to the squashed commit when we merge the PR, so you both get credit for the contribution.
> 
> https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Sounds good, I can combine our changes. I can't seem to find @ddouglas 's email address to add to the commit. Please let me know if the user name will suffice in the commit message. Thanks!

File: README.md [hunk 2]
```diff
@@ -31,21 +31,28 @@ This will also clone and initialize all of the submodules that this project depe
...
 
 You will need to install the [Emscripten toolchain](https://emscripten.org) first.
 
-	% mkdir build-web
-	% cd build-web
-	% emcmake cmake ..
-	% cmake --build .
-	% emrun ./raven.html
+```shell
+  mkdir build-web
```
Comment (jminor) [inline README.md:50]: Could you add the `git submodule update` command here also, in case someone starts with the emscripten build?

Comment (yaash45) [reply README.md:51]: Good catch!

**Pull request merged**